database/schedule/update.go

```diff
@@ -13,7 +13,7 @@ import (
 )
 
 // UpdateSchedule updates an existing schedule in the database.
-func (e *engine) UpdateSchedule(s *library.Schedule) error {
+func (e *engine) UpdateSchedule(s *library.Schedule, config bool) error {
```

Comment:
Thoughts on renaming this to `fields`? Or a different name?

```suggestion
func (e *engine) UpdateSchedule(s *library.Schedule, fields bool) error {
```

I could see a scenario where `config` may not be clear on what changes it will drive.